src/keylessBackup/SignInWithEmail.tsx

```diff
@@ -176,10 +176,10 @@ function SignInWithEmail({ route }: Props) {
     }
   }
 
-  if (!address) {
+  if (!address && isSetup) {
```

Comment:
should this be `!isSetupInOnboarding` instead?

Reply:
I think the only thing that using `!isSetupInOnboarding` would change is that we would _not_ spin if the user is setting up CAB from the settings screen but their wallet address doesn't exist, which, while it seems impossible, is probably a better experience? i can make that change if we like

Reply:
KK, i've made the change here and in the [shefi PR](https://github.com/mobilestack-xyz/mobilestack-shefi/pull/16).

Reply:
oooh i see, i misunderstood the meaning of the name. i see that the `keylessBackupFlow` value is "restore" for the onboarding restore flow. as long as this also works for the onboarding flow for making a new wallet, then all good (naively i thought that we would still be in the `!address && isSetup` condition in that case)